github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-presubmits.yaml

```diff
@@ -419,16 +419,9 @@ presubmits:
             - "-c"
             - |
               cd images
-              ./publish_image.sh -b pr-creator quay.io kubevirtci
+              ./publish_multiarch_image.sh -b pr-creator quay.io kubevirtci
           securityContext:
             privileged: true
-          resources:
-            requests:
-              memory: "16Gi"
```

Comment:
Can you please just re-add the memory requests here ? This just ensures that is gets enough RAM on the node.

Reply:
> @brianmcarey You mean only memory requests ie., ` resources: requests: memory: "16Gi"`
> 
> without adding back the cpu requests and memory+cpu limits ?
> 
Yes without the cpu requests and memory+cpu limits - thanks

Reply:
> 
> P.S: As we discussed earlier in DM, I will add -a flag to ./publish_multiarch_image.sh once testing is complete as this is pre-submit job.

I am ok with leaving the presubmit as is for now - this image doesn't get updated that often so doing the full build of the three images in the presubmit should not be too costly.

Reply:
Better I will add -a. That way I can keep the same resource requests 16Gi, than changing to 29Gi, as multi-arch build for all archs needs more resources. WDYT?

Reply:
For now I'm leaving this as is (without -a) and adding 29Gi for memory.

Later before merge I can change - to add -a and keeping only memory request to 16Gi.